api-reference/10 UI Components/dxDiagram/1 Configuration/customShapes/textWidth.md

```diff
@@ -9,4 +9,6 @@ Specifies the shape text container's width.
 ---
 This property specifies a fraction from 0 to 1 that represents the shape text's width in relation to the shape width. If the **textWidth** property is set to 1, the text width is equal to the shape width.
```

Comment:
`This property specifies a fraction from 0 to 1 that represents the shape text's width in relation to the shape width.`

Please rewrite without using "represents".

Reply:
fixed